tests/pytest/test_multi.py

```diff
@@ -489,14 +509,25 @@ def testArrPopCommand(env):
     res = r.execute_command('JSON.GET', 'doc1', '$')
     r.assertEqual(json.loads(res), [{"a": [], "nested1": {"a": ["hello", "world"]}, "nested2": {"a": 31}}])
     # Test single
-    res = r.execute_command('JSON.ARRPOP', 'doc1', '.nested1.a', -2, '"baz"')
+    res = r.execute_command('JSON.ARRPOP', 'doc1', '.nested1.a', -2)
```

Comment:
Can you explain this change?

Reply:
This is a superfluous argument
(we could consider adding `args.done()?;` to ensure no unnecessary args are supplied)